pyInterface/CMakeLists.txt

```diff
@@ -162,12 +162,12 @@ if(PYLINT_FOUND)
 	# check 'pyRootPwa' module
 	add_test(
 		NAME pyLintPyRootPwa
-		COMMAND ${PYLINT_EXECUTABLE} --reports=n --rcfile=${CMAKE_CURRENT_SOURCE_DIR}/pylintrc pyRootPwa
+		COMMAND ${PYLINT_EXECUTABLE} --reports=n --rcfile=${CMAKE_CURRENT_SOURCE_DIR}/pylintrc module
 	)
 	set_property(
 		TEST pyLintPyRootPwa
 		APPEND
-		PROPERTY ENVIRONMENT "PYTHONPATH=${PYTHON_LIBRARY_OUTPUT_PATH}:$ENV{PYTHONPATH}"
+		PROPERTY ENVIRONMENT "PYTHONPATH=${CMAKE_CURRENT_SOURCE_DIR}:${PYTHON_LIBRARY_OUTPUT_PATH}:$ENV{PYTHONPATH}"
```

Comment:
It seems we do not need the `${PYTHON_LIBRARY_OUTPUT_PATH}` here at all.

Reply:
Is is required, e.g. if ROOTPWA is not in the PYTHONPATH or another ROOTPWA version is in the PYTHONPATH.